packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6114,7 +6132,6 @@ export default class Meeting extends StatelessWebexPlugin {
 
     if (content && this.mediaProperties.mediaDirection.sendShare) {
       Metrics.postEvent({event: eventType.SHARE_STOPPED, meeting: this});
-      Media.stopTracks(this.mediaProperties.shareTrack);
```

Comment:
do we not need to close the new track?
`this.setLocalShareTrack(null)` maybe?

Reply:
I had to move this out, because webrtc-core emits the "ended" event when you call `stop()` on a track (that's different to how normal webrtc tracks work - they don't emit "ended" when you call `stop()` on them) so without this change, I had an infinite loop when releasing the floor, because here we were doing stop which was emitting "ended" which was triggering release of the floor, so calling stop again etc.
And anyway I think it's better to not call stop() here, because this is a `releaseScreenShareFloor()` method, so all it should do is just release the floor with Locus and not do anything else. Stopping of the track is done outside of it - either just before or after it's called.